examples/plot_flux.py

```diff
@@ -0,0 +1,18 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import openmc
+
+mpl.use("TkAgg")
+
+# Get the flux from the statepoint
+with openmc.StatePoint("statepoint.10.h5") as sp:
+    flux = np.log(sp.tallies[1].mean)
```

Comment:
optional idea, nice to get the tally by score to help show linkage to the previous script
```suggestion
flux = np.log(sp.get_tally(scores=['flux']).mean)
```